### ISSUE TITLE
Feature request: avoid auto-tagging users who have used 'mute conversation'

### ISSUE THREAD

**Issue by strypey** [tootsuite/mastodon#13205]
I'm involved in a thread with about 10 people. As with any thread with that many participants that goes on for long enough, it's going off on various tangents from the original conversation that brought everyone in. Some of the participants may have untagged themselves by using 'mute conversation' at their end, but their addresses are still taking up chars in the text field of each post in the thread. This forces responses to be terser that they might actually need to be to include everyone (forced brevity can increase the chances of miscommunication), as well as making it difficult for new people to enter the conversation if they have something to contribute.

Is there any way for a Mastodon client, when it responds to a user clicking 'reply', to detect when a user has used 'mute conversation' and avoid copying their address from the previous post into the reply text field? 

I first asked @nolanlawson about this on the [Pinafore issues board](https://github.com/nolanlawson/pinafore/issues/1682#issuecomment-569440600). He replied:
> As far as I know, there's nothing in the Mastodon API that allows me to query whether a user has muted a certain conversation

**Comment by ThibG** [comment]
No, there is nothing in the API for this, and it's not federated information either, so if we did something, that would only work if both people were on the same instance.
In general, I'm very uneasy with making people aware they were muted (even if that's within the context of one specific conversation)

**Comment by Cassolotl** [comment]
>In general, I'm very uneasy with making people aware they were muted (even if that's within the context of one specific conversation)

I came in here to say this!

**Comment by strypey** [comment]
@ThibG wrote:
> No, there is nothing in the API for this, and it's not federated information either

OK, so this is the feature I'm requesting. Ideally it would work across all AP networks, not just on Mastodon instances.

> I'm very uneasy with making people aware they were muted

I hadn't thought of that and I totally understand the concern. To be clear, I'm asking for this only in the context of the 'mute conversation' feature, not 'mute user'. 

> even if that's within the context of one specific conversation

In my experience, when 'mute conversation' is used instead of 'mute user', it's because none of the users in the conversation are Bad Actors (or they'd be muted or blocked individually), but the user muting the conversation has lost interest in it and doesn't want it cluttering up their @mentions.

Another solution to the same problem would be to modify the code that implements the 500 char limit on posts so that it no longer include any @user@domain.foo text.
